Add support for gfx1030
Install executables in bin subdirectory
Allow roctracer to be installed in a separate location
diff --git a/CMakeLists.txt b/CMakeLists.txt
index c1c8704..48489a7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -110,7 +110,7 @@ if(HIP_VERSION VERSION_GREATER_EQUAL "4.3")
   set(TARGETS "${TARGETS};gfx90a")
 endif()
 if (HIP_VERSION VERSION_GREATER_EQUAL "5.3")
-  set(TARGETS "${TARGETS};gfx940")
+  set(TARGETS "${TARGETS};gfx940;gfx1030")
 endif()
 if (HIP_VERSION VERSION_GREATER_EQUAL "5.7")
   set(TARGETS "${TARGETS};gfx941;gfx942")
@@ -195,9 +195,9 @@ file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/scripts/HPL.dat
 rocm_install_targets(TARGETS rochpl)

 install(PROGRAMS ${CMAKE_BINARY_DIR}/run_rochpl ${CMAKE_BINARY_DIR}/mpirun_rochpl
-        DESTINATION ${CMAKE_INSTALL_PREFIX})
+        DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
 install(FILES ${CMAKE_BINARY_DIR}/HPL.dat
-        DESTINATION ${CMAKE_INSTALL_PREFIX})
+        DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)

 # Package specific CPACK vars
 set(CPACK_DEBIAN_PACKAGE_DEPENDS "rocm-dev (>= 3.5.0)")
diff --git a/cmake/Dependencies.cmake b/cmake/Dependencies.cmake
index 2154faf..cb824f0 100644
--- a/cmake/Dependencies.cmake
+++ b/cmake/Dependencies.cmake
@@ -83,10 +83,10 @@ list(APPEND CMAKE_PREFIX_PATH ${ROCBLAS_PATH} ${ROCM_PATH} )
 list(APPEND CMAKE_MODULE_PATH ${ROCM_PATH}/lib/cmake/hip )

 find_library(ROCTRACER NAMES roctracer64
-             PATHS ${ROCM_PATH}/lib
+             PATHS ${ROCTRACER_PATH}/lib
              NO_DEFAULT_PATH)
 find_library(ROCTX NAMES roctx64
-             PATHS ${ROCM_PATH}/lib
+             PATHS ${ROCTRACER_PATH}/lib
              NO_DEFAULT_PATH)

 message("-- roctracer:  ${ROCTRACER}")
@@ -94,13 +94,13 @@ message("-- roctx:      ${ROCTX}")

 add_library(roc::roctracer SHARED IMPORTED)
 set_target_properties(roc::roctracer PROPERTIES
-  INTERFACE_INCLUDE_DIRECTORIES "${ROCM_PATH}/include"
+  INTERFACE_INCLUDE_DIRECTORIES "${ROCTRACER_PATH}/include"
   INTERFACE_LINK_LIBRARIES "hip::host"
   IMPORTED_LOCATION "${ROCTRACER}"
   IMPORTED_SONAME "libroctracer.so")
 add_library(roc::roctx SHARED IMPORTED)
 set_target_properties(roc::roctx PROPERTIES
-  INTERFACE_INCLUDE_DIRECTORIES "${ROCM_PATH}/include"
+  INTERFACE_INCLUDE_DIRECTORIES "${ROCTRACER_PATH}/include"
   INTERFACE_LINK_LIBRARIES "hip::host"
   IMPORTED_LOCATION "${ROCTX}"
   IMPORTED_SONAME "libroctx64.so")
